cookbooks/travis_internal_base/recipes/default.rb

```diff
@@ -40,10 +39,17 @@
   only_if { File.directory?('/etc/cloud') }
 end
 
-cookbook_file '/usr/local/bin/generate-ssh-host-keys' do
-  owner 'root'
-  group 'root'
-  mode 0o755
+%w(
+  generate-ssh-host-keys
+  set-hostname-from-template
+  create-users
+).each do |script|
+  cookbook_file "/var/lib/cloud/scripts/per-instance/#{script}" do
```

Comment:
That's correct, as long as we have `scripts-per-instance` somewhere in `cloud_init_modules`, `cloud_config_modules`, or `cloud_final_modules`.